staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -61,16 +63,31 @@ func SchemaDeclType(s *schema.Structural, isResourceRoot bool) *DeclType {
 	case "array":
 		if s.Items != nil {
 			itemsType := SchemaDeclType(s.Items, s.Items.XEmbeddedResource)
+			// TODO(DangerOnTheRanger): calculate maxLength if not given in the schema
+			// TODO(DangerOnTheRanger): refactor -1 (indicating no length given) out into a constant
```

Comment:
Since all arrays, maps and strings will end up being assigned a limit. Do you need an extracted constant?

Reply:
That's true, when every CEL-related PR merges a constant wouldn't make much sense. Maybe it'd be fine to leave it as-is because of that.